Lib/test/test_sys_settrace.py

```diff
@@ -644,16 +644,22 @@ def func():
                 4
             else:
                 6
+                if False:
```

Comment:
Does this cover the original issue?
The compiler will optimize out the `if False`.
Might be a good idea to add another test with a non-constant predicate.

Reply:
The test did fail without the fix. But sure, I'll add a test with a non const.